sky/engine/bindings/jni/jni_array.cc

```diff
@@ -433,7 +433,7 @@ int64_t JniLongArray::GetArrayElement(jsize index) {
   return false;
 }
 
-void JniLongArray::SetArrayElement(jsize index, int64_t value) {
+void JniLongArray::SetArrayElement(jsize index, jlong value) {
```

Comment:
Same here - take an int64_t argument, and static_cast it to a jlong before calling the JNI API.
(see some of the SetArrayElement implementations for other Java types)